applications/FluidDynamicsApplication/tests/adjoint_vms_element_2d.py

```diff
@@ -253,4 +263,10 @@ def test_CalculateSensitivityMatrix(self):
         self._assert_matrix_almost_equal(FDShapeDerivativeMatrix, ShapeDerivativeMatrix)
 
 if __name__ == '__main__':
-    KratosUnittest.main()
+    test = AdjointVMSElement2D()
```

Comment:
Why did you switch from `KratosUnittest.main()`? I liked the old style printout more because it tells you information about the tests that run. This information is missing now.

Reply:
I changed this in few tests, but some of the tests in FluidDynamicsApplication has some specific method calls. Therefore those are not changed.